src/content/pages/explore.mdx

```diff
@@ -0,0 +1,144 @@
+- Kunratický les: forest
+- Hostivář: area for walk, you can swim there as well
+
+*Use [mapy.cz](https://mapy.cz/) - there are routes there (red, blue and green ones) - the best maps for hiking in general*
+
+## Where to eat?
+
+Prague is a food city! There are many places to find nice food, from traditional Czechia food to Vegan friendly food.
+
+### General recommendation
+
+Some yays and nays from our local volunteers:
+
+- 👎 Trdelnik is a tourist thing: nothing original or authentic
+- 👎 Pražská šunka on the street: tourist trap
+- 👍 Try Nakládaný hermelín (pickled camembert)
+- 👍 Smažený sýr (fried cheese)
+- 👍 Svíčková (meat with vegetable sauce and dumplings)
+- 👍 Guláš (goulash)
+
+
+### For veggie/ vegans
+
+We know some of us are veggie / vegan. Don't worry, you will find places that serve you veggie / vegan food:
+
+- Shromazdiste: typical czech food in vegan versions, great food and great beer
+- Eateria: vegan junk food, best burger
+- Pastva: open during day only, european and healthy food
+- Dhaba beas: cheap buffet, indian and european food, it’s everywhere
+- Share Sweet & Espresso Bar: coffee shop with best vegan cakes
+
+According to HappyCow there are 89 vegan and vegetarian places to eat. So it should not be an issue to find something. Typical Czech restaurants mostly only offer fried cheese and french fries, sometimes pickled camembert.  Czech cuisine is mostly meat-heavy.
+Hungry during the night? Go to any kebab place and you can order falafel wrap with veggies. You can also go to Burrito Loco - it’s a mexican food chain and they offer vegan mana burrito and burrito with grilled veggies.
+
+[Visit happy cow vegan-vegetarian listings](https://www.happycow.net/europe/czech_republic/prague/?filters=vegan-vegetarian)
+
+## Where to drink?
+
+In European counties drinking is a popular way to socialise. Here are some recommendations:
+
+- For craft beer: Sibeeria bar, Pivotéka, BeerGeek, Zlý časy
+- Pilsner type beer: everywhere
+- For mild and not bitter Pilsner-style beer try Únětice (it’s in a few places), they also have 8 degrees with less alcohol
+- For more bitter try Plzeň/Pilsner or Kozel, Radegast
+For craft try Matuška (APA, IPA, ALE), Zíchovec (Sour ALE, APA, IPA, ALE) or Sibeeria (Sour ALE, APA, IPA, ALE)
+- For wine: Bublinka 2024 wine festival
+
+On top of the above:
+
+### All day social places
+
+- Automat Matuška
+
+### Recommendation for music lovers
+
+There are multiple festivals going on you might want to check out:
+
+- Jazz dock for jazz on a boat
+- Lucerna
+- Prague Open Air festival
+- Prague Rooftop festival
+- Letní scéna Musea Kampa 2024 festival music + theater
+- Jazz festival Mladí ladí jazz 2024
+
+[You can find more events here](https://goout.net/en/prague/events/leznyvlkk/?after=2024-07-08%2000%3A00&before=2024-07-14%2023%3A59)
+
+### For party animals and clubbing enthusiasts
+
+- Roxy: all genres
+- Lucerna: pop, sometimes alternative
+- Cross club: punk, alternative
```

Comment:
Ran the whole thing through ChatGPT to fix grammar mistakes

```suggestion
---
title: More Tips on Exploring Prague
subtitle: At EuroPython, many attendees will come with their families. While not attending the conference, why not go and explore Prague and the surrounding area?
---

import { Image } from "astro:assets";
import venueImage from "./images/prague.jpg";

# More Tips on Exploring Prague

<Image src={venueImage} width="1200" alt="" />

<br></br>

At EuroPython, many attendees will come with their families. While not attending the conference, why not go and explore Prague and the surrounding area?

Here are some tips and things to do provided by our volunteers. We hope they help you and your companions have a great time in Prague:

## Traveling around

### Public transport

In Prague, the recommended way to get around is public transport:

- There’s some info [here](https://ep2024.europython.eu/where#getting-around-in-prague)
- You can use your normal card to travel around by train within Prague
- It is usually fast and reliable
- After midnight, the metro stops, but there are buses
- Mostly safe, with no no-go stations or anything similar

### Do I need to rent a car?

Some people may consider renting a car, but:

- In Prague, this is unnecessary. It’s hard to find parking, and it’s expensive
- You can rent a car at the airport or other places in the city (like you normally do in other cities)
- If you want to visit other towns, you can use trains as well (they’re mostly cheap)

### Be prepared for the weather

If you want to explore the city, you will be exposed to the weather. So here are some answers to frequent questions like: What’s the weather like? What should you carry with you?

- Usually 30°C+, so carry water
- There are no water fountains or public toilets, so carry more water
- Use sunscreen
- It rarely rains, but when it does, it’s shower rain. It’s best to just hide somewhere until it stops (it usually stops after half an hour or so).

## What to do while in Prague?

There are plenty of things you can do in Prague. Depending on your preferences, you can find things that interest you and your group.

### For culture lovers

- Summer cinema
- Theater festival Za dveřmi 2024

### For art lovers

- DOX
- Soví mlýny
- National Museum
- Jazz Dock for live jazz on a boat
- Žižkov area for bars and pubs
- House of the Black Madonna: cafe shop in a building that’s built in cubist style (there was a cubist architecture movement in Prague and it’s quite famous and unique)

### For nature lovers

- Petřín Hill: Eiffel Tower (in Prague), nice view of the city
- Vyšehrad: nice fortress next to the venue. It also features a park where you can sit and chill
- Letná: viewpoint
- Riegrovy Sady: park with a nice view. There’s a rose garden most people don’t know about - you should check it out
- Divoká Šárka: small forest a bit further away, you can swim in Džbán (an artificial lake)
- Kunratický Les: forest
- Hostivář: area for walking, you can swim there as well

*Use [mapy.cz](https://mapy.cz/) - there are routes there (red, blue, and green ones) - the best maps for hiking in general*

## Where to eat?

Prague is a food city! There are many places to find nice food, from traditional Czech food to vegan-friendly options.

### General recommendations

Some yays and nays from our local volunteers:

- 👎 Trdelnik is a tourist thing: nothing original or authentic
- 👎 Pražská šunka on the street: tourist trap
- 👍 Try Nakládaný Hermelín (pickled camembert)
- 👍 Smažený sýr (fried cheese)
- 👍 Svíčková (meat with vegetable sauce and dumplings)
- 👍 Guláš (goulash)

### For veggies/vegans

We know some of us are veggies/vegans. Don't worry, you will find places that serve you veggie/vegan food:

- Shromáždiště: typical Czech food in vegan versions, great food, and great beer
- Eateria: vegan junk food, best burger
- Pastva: open during the day only, European and healthy food
- Dhaba Beas: cheap buffet, Indian and European food, it’s everywhere
- Share Sweet & Espresso Bar: coffee shop with the best vegan cakes

According to HappyCow, there are 89 vegan and vegetarian places to eat. So it should not be an issue to find something. Typical Czech restaurants mostly only offer fried cheese and French fries, sometimes pickled camembert. Czech cuisine is mostly meat-heavy. Hungry during the night? Go to any kebab place and you can order a falafel wrap with veggies. You can also go to Burrito Loco - it’s a Mexican food chain and they offer vegan mana burrito and burrito with grilled veggies.

[Visit HappyCow vegan-vegetarian listings](https://www.happycow.net/europe/czech_republic/prague/?filters=vegan-vegetarian)

## Where to drink?

In European countries, drinking is a popular way to socialize. Here are some recommendations:

- For craft beer: Sibeeria Bar, Pivotéka, BeerGeek, Zlý časy
- Pilsner-type beer: everywhere
- For mild and not bitter Pilsner-style beer, try Únětice (it’s in a few places). They also have 8 degrees with less alcohol
- For more bitter, try Plzeň/Pilsner or Kozel, Radegast
For craft, try Matuška (APA, IPA, ALE), Zíchovec (Sour ALE, APA, IPA, ALE) or Sibeeria (Sour ALE, APA, IPA, ALE)
- For wine: Bublinka 2024 wine festival

On top of the above:

### All-day social places

- Automat Matuška

### Recommendations for music lovers

There are multiple festivals going on you might want to check out:

- Jazz Dock for jazz on a boat
- Lucerna
- Prague Open Air Festival
- Prague Rooftop Festival
- Letní scéna Musea Kampa 2024 Festival: music + theater
- Jazz Festival Mladí ladí jazz 2024

[You can find more events here](https://goout.net/en/prague/events/leznyvlkk/?after=2024-07-08%2000%3A00&before=2024-07-14%2023%3A59)

### For party animals and clubbing enthusiasts

- Roxy: all genres
- Lucerna: pop, sometimes alternative
- Cross Club: punk, alternative
```

Reply:
I am not against chat GPT but I think we need at least a pair of human eyes to final review it. I cannot see the preview generated, @egeakman can you help? Just check if chat GPT says anything that doesn't make sense

Reply:
Hey @cheukting, of course! I'll go over it tomorrow.

Reply:
@Cheukting I wonder if it is easier to accept the suggestions and then I am happy to review from there. It is a bit difficult now to know what has already been suggested. 
P.S. looking forward to visiting so many of these wonderful places <3 
P.P.S: I have a pair of human eyes! ☺️

Reply:
@hypha @Cheukting Ah I completely forgot this, sorry! I've gone through it; the chatgpt suggestion looks good overall, but it misses the Czech accents time to time. Like ``í`` in ``Trdelník``. Other than this, I agree with Raquel's suggestion [below](https://github.com/EuroPython/website/pull/690#pullrequestreview-2127767339).

Reply:
I don't think missing accents is an issue 🙂 If you need me to add them, feel free to commit or merge it and I'm happy to go over it and add accents, where missing 🙂

Reply:
Is it OK to just commit them and then we can quickly do a final pass?

Reply:
If it's OK with @Cheukting, we can commit the suggestion.